src/explorer/TimelineCredViewContainer.js

```diff
@@ -30,10 +31,11 @@ const DEFAULT_ENTRIES_PER_CHART = 6;
  * and then navigating to:
  * http://localhost:8080/test/TimelineCredView/
  */
-export class TimelineCredView extends React.Component<Props> {
+export class TimelineCredViewContainer extends React.Component<Props> {
```

Comment:
I think `TimelineCredViewContainer` is a confusing name. As it's a React component, the concept of `Container` would (IMO) more naturally apply to the React components that it contains, so in that sense it contains a `TimelineCredChart`, not a `TimelineCredView`.

How about we call it a `TimelineCredRenderer`?